ADR/Components/Aerodynamic_components/tests/test_Airfoil.py

```diff
@@ -0,0 +1,20 @@
+from ADR.Components.Aerodynamic_components.Airfoil import *
+import numpy.testing as npt
+
+
+def test_get_Cl():
+    aircraft_airfoil_data = {"airfoil_name": "s1223"}
+    aircraft_Coefficients = Airfoil(aircraft_airfoil_data)
```

Comment:
This is an instance of the Airfoil class, so "airfoil" would better explain what it is.